### ISSUE TITLE
TypeConverterManager "require is not defined"

### ISSUE THREAD

**Issue by johnnyd710** [iTwin/appui#465]
**Describe the bug**
The `TypeConverterManager.ts` file uses a dynamic require in the esm output, which causes problems when building with Vite: `require is not defined`

```js
      } = require("../converters/StringTypeConverter"); // eslint-disable-line @typescript-eslint/no-var-requires
```

**To Reproduce**
Difficult to reproduce... create a sample app using Vite, use the property grid widget, but to hit the error inside the if condition depends on the iModel or element selected. As soon as the require is hit, the error pops up because the require() method is supported in browsers

**Screenshots**
![image](https://github.com/iTwin/appui/assets/19596966/da1dcb0e-5182-4527-99fc-cea03af57475)

**Additional context**
If I use vite/rollup's [transformMixedEsModules](https://www.npmjs.com/package/@rollup/plugin-commonjs#transformmixedesmodules), the require statement is successfully transformed, but then I get a different error `Cannot access 'TypeConverterManager' before initialization at StringTypeConverter.js:92:1`. I believe the new issue is caused by a circular dependency between TypeConverterManager and StringTypeConverter.

**Comment by raplemie** [comment]
This was fixed in 3.x but we missed the 4.0 backport, I'll look into it.

**Comment by raplemie** [comment]
A fix have been done, I'll let you know when it gets released.

**Comment by johnnyd710** [comment]
> A fix have been done, I'll let you know when it gets released.

Awesome, thanks for the quick fix.

**Comment by raplemie** [comment]
Hi @johnnyd710, a fix for this was released today in 4.5.0. Please try it out and let us know if it fixes your issue!
